### PR TITLE
addnewplaceseed

### PR DESCRIPTION
**Pull request by Ilhembsk**
I added places in the seeds.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -73,41 +73,151 @@
...
+                  #rating: rand(1..5),
+                  #senses: Place::SENSES.sample,
+                  #exterior: false,
+                  #environment: Place::ENVIRONMENTS.sample,
+                  #description: Faker::Quote.jack_handey
+                #)
+ #place.photo.attach(io: file, filename: 'file.png', content_type: 'image/png')
+  #place.save
+
+p1= Place.new(name: Musée Cantini,
```
Comment (skhohar) [inline db/seeds.rb:101]: I believe names must be strings

File: db/seeds.rb [hunk 2]
```diff
@@ -168,6 +278,140 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (skhohar) [inline db/seeds.rb:415]: still needs : 
- associate the right picture to each place
- doing a for each on an array of places in order to save them

**Review by skhohar — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (Ilhembsk) [reply db/seeds.rb]: Done

File: db/seeds.rb (hunk 2 above)
Comment (Ilhembsk) [reply db/seeds.rb:485]: done

**Pull request merged**